src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -165,6 +170,14 @@ function WorkspacesListPage({policies, allPolicyMembers, reimbursementAccount, r
                 });
             }
 
+            if (!(isAdmin || isOwner)) {
+                threeDotsMenuItems.push({
+                    icon: Expensicons.ChatBubbles,
```

Comment:
Hi 👋 coming from https://github.com/Expensify/App/issues/46248.

Pretty minor issue and part of the polish issue. We should be using the `Exit` icon.